src/main/java/shapestone/jdbc/mysql/Connections.java

```diff
@@ -0,0 +1,198 @@
+	}
+
+	public static void createTable() throws SQLException {
+		Connection connect = getConnection();
+
+		Statement statement = connect.createStatement();
+
+		String query = "DROP TABLE IF EXISTS patients";
+		String query1 = "DROP TABLE IF EXISTS treatments";
+		statement.execute(query1);
+		statement.execute(query);
+
+		// Create 'patients' table
+		StringBuilder patientsTableQuery = new StringBuilder("CREATE TABLE patients (");
+		patientsTableQuery.append("patientId INT PRIMARY KEY, ");
+		patientsTableQuery.append("name VARCHAR(50) NOT NULL, ");
+		patientsTableQuery.append("age INT NOT NULL, ");
+		patientsTableQuery.append("gender VARCHAR(10) NOT NULL, ");
+		patientsTableQuery.append("address VARCHAR(100) NOT NULL");
+		patientsTableQuery.append(")");
+		statement.execute(patientsTableQuery.toString());
+
+		// Create 'treatments' table
+		StringBuilder treatmentsTableQuery = new StringBuilder("CREATE TABLE treatments (");
+		treatmentsTableQuery.append("treatmentId INT PRIMARY KEY, ");
+		treatmentsTableQuery.append("patientId INT NOT NULL, ");
+		treatmentsTableQuery.append("diagnosis VARCHAR(50) NOT NULL, ");
+		treatmentsTableQuery.append("treatment VARCHAR(50) NOT NULL, ");
+		treatmentsTableQuery.append("dateOfTreatment DATE NOT NULL, ");
+		treatmentsTableQuery.append("cost DECIMAL(10, 2) NOT NULL, ");
+		treatmentsTableQuery.append("FOREIGN KEY (patientId) REFERENCES patients(patientId)");
+		treatmentsTableQuery.append(")");
+		statement.execute(treatmentsTableQuery.toString());
+
+		System.out.println("Both tables are created successfully.");
+
+	}
+
+	public static void insertData(List<PatientPojo> patientList, List<TreatmentPojo> treatmentList)
+			throws SQLException {
+		Connection connect = getConnection();
+		String sql = "INSERT INTO patients (name, patientId, age, gender, address) VALUES (?, ?, ?, ?, ?)";
+		String sqll = "INSERT INTO treatments(treatmentId,patientId,diagnosis,treatment,dateOfTreatment,cost) VALUES (?, ?, ?, ?, ?, ?)";
+		PreparedStatement statement = connect.prepareStatement(sql);
+		PreparedStatement statementt = connect.prepareStatement(sqll);
+		for (PatientPojo patient : patientList) {
+			statement.setString(1, patient.getName());
+			statement.setInt(2, patient.getpatientId());
+			statement.setInt(3, patient.getAge());
+			statement.setString(4, patient.getGender());
+			statement.setString(5, patient.getAddress());
+			statement.executeUpdate();
+		}
+		for (TreatmentPojo treatment : treatmentList) {
+			statementt.setInt(1, treatment.getTreatmentId());
+			statementt.setInt(2, treatment.getPatientId());
+			statementt.setString(3, treatment.getDiagnosis());
+			statementt.setString(4, treatment.getTreatment());
+			statementt.setString(5, treatment.getDateOfTreatment());
+			statementt.setDouble(6, treatment.getCost());
+			statementt.executeUpdate();
+		}
+
+		System.out.println("both tables data inserted successfully");
+	}
+
+	public static void readData() throws SQLException {
```

Comment:
what if I want to read particular patient data ??

I mean to pass patientId give an option, otherwise fetch all data